packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_blocs/coordinator_bloc.dart

```diff
@@ -17,11 +20,15 @@ abstract class CoordinatorEvents {
     required ErrorModel error,
     String? stackTrace,
   });
+  {{#enable_change_language}}
+  void setCurrentLanguage(LanguageModel model);{{/enable_change_language}}
 }
 
 abstract class CoordinatorStates {
   @RxBlocIgnoreState()
   Stream<bool> get isAuthenticated;
+  {{#enable_change_language}}
+  Stream<LanguageModel> get currentLanguage;{{/enable_change_language}}
```

Comment:
As discussed most probably this needs to be moved to the change language bloc

Reply:
These event `setCurrentLanguage` and state : `currentLanguage` were moved to the `ChangeLanguageBloc`